scripts/docker-compose/imports/smartdocuments-wiremock/__files/sdapi-structure-response-body.json

```diff
@@ -0,0 +1,227 @@
+{
+  "documentsStructure": {
+    "templatesStructure": {
+      "templateGroups": [
+        {
+          "templateGroups": [
+            {
+              "templateGroups": [],
+              "templates": [
+                {
+                  "favorite": false,
+                  "id": "7B7857BB9959470C82974037304E433D",
+                  "name": "Data Test"
```

Comment:
Minor thing but these test template names seem a little arbitrary and weird? I would rather name them something like `smartDocumentsTestTemplate1` and `smartDocumentsTestTemplate2` or something?

Reply:
Renamed to `root`, `group 2 template 1`, etc.